src/games/strategy/triplea/ai/proAI/util/ProSortMoveOptionsUtils.java

```diff
@@ -54,7 +54,8 @@ public int compare(final Map.Entry<Unit, Set<Territory>> o1, final Map.Entry<Uni
   }
 
   public static Map<Unit, Set<Territory>> sortUnitNeededOptions(final PlayerID player,
-      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap) {
+      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap,
```

Comment:
One thing to mention. To some extent every extra thing you can do is very efficient. Both of us took time to gain context into this code. If you gain context and clean something that is hard to understand, that is roughly 10 times you never have to relearn that chunk of code, and multiplied by N people - it's a very big savings. So really teh issue is, yes it's out of scope and extra, but the extra 5 minutes you take after having already payed the context cost, saves 30 minutes for many other people repeatedly. We do agree that at some point you do need to call it a day and not refactor the whole code base. Yet, the time savings of having context and cleaning things up is also very important too.